speech/cloud-client/README.rst

```diff
@@ -230,6 +230,7 @@ To run this sample:
         python beta_snippets.py enhanced-model resources/commercial_mono.wav
         python beta_snippets.py metadata resources/commercial_mono.wav
         python beta_snippets.py punctuation resources/commercial_mono.wav
+        python beta_snippets.py multi-language resources/commercial_mono.wav
```

Comment:
this, and the changes to the beta_snippets.py file might cause conflicts against the other PRs.